modules/cudaoptflow/src/pyrlk.cpp

```diff
@@ -53,6 +53,21 @@ Ptr<cv::cuda::DensePyrLKOpticalFlow> cv::cuda::DensePyrLKOpticalFlow::create(Siz
 
 #else /* !defined (HAVE_CUDA) */
```

Comment:
`throw_no_cuda()` implementation should be added.

Reply:
I think `throw_no_cuda` is covered by, `#if !defined (HAVE_CUDA) || defined (CUDA_DISABLER)`. May be, `!` is a typo in `#else /* !defined (HAVE_CUDA) */`. Isn't it?